csi.proto

```diff
@@ -635,6 +671,25 @@ message NodeUnpublishVolumeRequest {
   string target_path = 2;
 }
 
+message NodeExpandVolumeRequest {
```

Comment:
should `NodeExpandVolumeRequest` contain a `volume_attributes` field?

Reply:
For now, I don't think so. Most SP's will call resize on volume path on node and that is the only information they would need. Is there an use case you were thinking?